tensorflow/python/kernel_tests/array_ops_test.py

```diff
@@ -572,9 +572,13 @@ def eval_if_tensor(x):
 
 STRIDED_SLICE_TYPES = [
     dtypes.int32, dtypes.int64, dtypes.int16, dtypes.int8, dtypes.float32,
-    dtypes.float64, dtypes.complex64, dtypes.complex128, dtypes.bool
+    dtypes.bool
 ]
 
+# DML doesn't support float64 and complex types
+if test_util.gpu_device_type() != "DML":
```

Comment:
"DML doesn't support float64"

Note, Slice will very soon :b (float64 is not special, just treated as opaque bits to memmove).

https://dev.azure.com/microsoft/WindowsAI/_git/WindowsAI/pullrequest/6101182?_a=files&path=%2Fdml%2FSharedValidation%2FDmlScatterOperatorValidator.h